strictdoc/export/html/generators/view_objects/diff_screen_view_object.py

```diff
@@ -39,27 +41,27 @@ def __init__(
         self.is_running_on_server: bool = project_config.is_running_on_server
         self.strictdoc_version = __version__
 
-    def render_screen(self, jinja_environment: JinjaEnvironment):
+    def render_screen(self, jinja_environment: JinjaEnvironment) -> Markup:
         return jinja_environment.render_template_as_markup(
             "screens/git/index.jinja", view_object=self
         )
 
-    def render_url(self, url: str):
-        return self.link_renderer.render_url(url)
+    def render_url(self, url: str) -> Markup:
+        return Markup(self.link_renderer.render_url(url))
 
-    def render_node_link(self, incoming_link, document, document_type):
+    def render_node_link(self, incoming_link, document, document_type) -> str:
```

Comment:
Could you explain why some render methods are Markup and some are str?

Is there a way we could unify this?

Reply:
Honestly I simply copied the signatures from document_screen_view_object.py because I remember back then I had a reason for `-> str`. But now I don't remember, so it's good you ask, we should reconsider. You could help me by telling when this method is invoked at all. I searched but didn't find a caller.

Reply:
1) I was wondering if all these render* methods should return `Markup`? If there is no specific reason to ever have `str`, we would have it consistent.

2) `render_node_link` could be a copy-and-paste from the other view object methods. Just remove it because I also don't see it being used.

Reply:
Actually, let me merge this one, and let's handle the str/Markup/unused stuff separately.